### ISSUE TITLE
spices-notifier@germanfr - 3.8 only?

### ISSUE THREAD

**Issue by smurphos** [linuxmint/cinnamon-spices-applets#1825]
* Applet version/Build date 3.8
 * Cinnamon version 3.6.7
 * Distribution - Mint 18.3
 * Graphics hardware *and* driver used
```
Card-1: Intel 2nd Generation Core Processor Family Integrated Graphics driver: i915 v: kernel bus ID: 00:02.0 
Display: x11 server: X.Org 1.18.4 driver: none unloaded: fbdev,intel,modesetting,vesa 
resolution: 1366x768~60Hz 
OpenGL: renderer: Mesa DRI Intel Sandybridge Mobile v: 3.3 Mesa 17.2.8 
direct render: Yes
```
**Notify author of applet**

@germanfr

**Issue**
Applet fails to load in Cinnamon 3.6.7. Unclear if this is a bug or the applet is for 3.8+ only.

Looking glass snippet below. No relevant errors apparent in xsession-errors.

```
error t=2018-05-14T07:21:46.032Z class is a reserved identifier
trace t=2018-05-14T07:21:46.032Z 
<----------------
Extension.prototype._init@/usr/share/cinnamon/js/ui/extension.js:156:13
Extension@/usr/share/cinnamon/js/ui/extension.js:115:5
loadExtension@/usr/share/cinnamon/js/ui/extension.js:450:25
init@/usr/share/cinnamon/js/ui/appletManager.js:47:9
start@/usr/share/cinnamon/js/ui/main.js:493:5
@<main>:1:31
---------------->
error t=2018-05-14T07:21:46.032Z [Applet "spices-notifier@germanfr"]: Error importing applet.js from spices-notifier@germanfr
error t=2018-05-14T07:21:46.032Z Could not load applet spices-notifier@germanfr
```

**Steps to reproduce**

Download and activate applet under Cinnamon 3.6.7

**Expected behaviour**

If by design applet in for Cinnamon 3.8+ only that should be specified in metadata.json - a note in the README would be good as-well ;-)

**Other information**

Just checked with Manjaro on Cinnamon 3.8+ and the applet works fine in that environment.

**Comment by germanfr** [comment]
It is written using the new class syntax, which wasn't supported in < 3.8. This is an applet for Cinnamon xlet developers and I'd expect them to have the latest version in a reasonable period of time, so I don't plan to do any backwards compatibility. That doesn't mean I will break things when new features are released, but I wrote it a few days ago under 3.8 already. However, you are free to patch it until you update your system. I guess you only need to change the classes to the old _init structure. Sorry.

**Comment by smurphos** [comment]
No worries. I guessed that was the case but just wanted to check. It's a very helpful tool BTW...thanks.
